plugin.video.vrt.nu/resources/lib/kodiwrappers/kodiwrapper.py

```diff
@@ -51,13 +51,14 @@ def show_listing(self, list_items, sort=None, content_type='episodes'):
         for title_item in list_items:
             list_item = xbmcgui.ListItem(label=title_item.title, thumbnailImage=title_item.art_dict.get('thumb'))
             url = self._url + '?' + urlencode(title_item.url_dict)
-            list_item.setProperty('IsPlayable', str(title_item.is_playable))
+            list_item.setProperty('isPlayable', 'true' if title_item.is_playable else 'false')
 
             if title_item.art_dict:
                 list_item.setArt(title_item.art_dict)
 
             if title_item.video_dict:
                 list_item.setInfo('video', infoLabels=title_item.video_dict)
+                list_item.setProperty('id', title_item.video_dict.get('videoid'))
```

Comment:
Is this useful?

Reply:
This PR is a Work-In-Progress, so I am just testing stuff.

Reply:
BTW I hoped this would somehow influence the "Watched" flag. I should probably start looking into the source code instead...